streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -1463,31 +1463,31 @@ public void adminClientMetricsVerification() {
 
         final StreamThread.StreamsMetricsThreadImpl streamsMetrics = new StreamThread.StreamsMetricsThreadImpl(metrics, "");
         final StreamThread thread = new StreamThread(
-                mockTime,
```

Comment:
Why we change the style here?

Reply:
I just did a copy-paste of the code and this is intellij auto formatting. Current style seems right to me.